lib/screen/home/start_workout.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:event_bloc/event_bloc.dart';
+import 'package:flutter/material.dart';
+
+class StartWorkout extends StatelessWidget {
```

Comment:
Usually I'd put buttons like this within the same file as the main screen. You don't have to do that for this instance. Just a note for when you'll create more Widgets and classes. Nothing makes me want to not work on something more than clicking on a folder and having 100 files in the same folder show up. Hahahaha